docs/react-v9/contributing/new-release-process-v9-packages.md

```diff
@@ -161,7 +161,16 @@ yarn nx g @fluentui/workspace-plugin:prepare-initial-release --project @fluentui
 > automation status: implemented ✅ / https://github.com/microsoft/fluentui/pull/28505
 
 ```sh
-yarn nx g @fluentui/workspace-plugin:prepare-initial-release --project @fluentui/react-one-preview --phase=stable
+yarn generate
```

Comment:
we should be consistent with previous steps

we can just update those commands that we have in place like

```diff
-yarn nx g @fluentui/workspace-plugin:prepare-initial-release --project @fluentui/react-one-preview --phase=preview
+yarn nx g @fluentui/workspace-plugin:prepare-initial-release --project react-one-preview --phase=preview
```

and additionally mention that one can use `yarn generate` or Nx Console to navigate through the process

WDYT?

Reply:
Added :)

Reply:
it seems we misunderstood what I was asking. 

- in this chapter, keep what we have just update the project name to not include `@fluentui/` -> `yarn nx g @fluentui/workspace-plugin:prepare-initial-release --project react-one-preview --phase=stable`
- before `## 1. Kick-off phase` chapter lets mention that "You can use `yarn generate` or Nx Console to navigate through the process.`"